x-pack/test/functional_with_es_ssl/apps/cases/group1/create_case_form.ts

```diff
@@ -126,5 +126,63 @@ export default ({ getService, getPageObject }: FtrProviderContext) => {
         await testSubjects.existOrFail('user-profile-assigned-user-cases_all_user2-remove-group');
       });
     });
+
+    describe('customFields', () => {
+      it('creates a case with custom fields', async () => {
+        const customFields = [
+          {
+            key: 'valid_key_1',
+            label: 'Summary',
+            type: CustomFieldTypes.TEXT,
+            required: true,
+          },
+          {
+            key: 'valid_key_2',
+            label: 'Sync',
+            type: CustomFieldTypes.TOGGLE,
+            required: true,
+          },
+        ];
+
+        await cases.api.createConfigWithCustomFields({ customFields, owner: 'cases' });
+
+        const caseTitle = 'test-' + uuidv4();
+        await cases.create.openCreateCasePage();
+
+        // verify custom fields on create case page
+        await testSubjects.existOrFail('create-case-custom-fields');
+
+        await cases.create.setTitle(caseTitle);
+        await cases.create.setDescription('this is a test description');
+
+        // set custom field values
+        const textCustomField = await find.byCssSelector(
+          `[data-test-subj*="text-create-custom-field"]`
+        );
+        await textCustomField.type('This is a sample text for summary!');
```

Comment:
super nit: if i understood this correctly should be `'This is a sample text for label!'` (and below too)